youtube_dl/extractor/vidlii.py

```diff
@@ -0,0 +1,136 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    get_element_by_id, get_element_by_class, strip_or_none,
+    float_or_none, urljoin, js_to_json, unified_strdate)
+
+
+class VidliiIE(InfoExtractor):
+    _VALID_URL = r'(?:https*?:\/\/)*(?:www\.)*vidlii.com\/watch\?v=(?P<id>[^?\s]{11})'
+    _TESTS = [{
+        'url': 'https://www.vidlii.com/watch?v=tJluaH4BJ3v',
+        'md5': '9bf7d1e005dfa909b6efb0a1ff5175e2',
+        'info_dict': {
+            'id': 'tJluaH4BJ3v',
+            'title': 'Vidlii is against me',
+            'description': 'I have HAD it. Vidlii does not like me. I have tried to uplaod videos and submit them to the '
+                           'contest and no ne of my videos show up so maybe it is broken for everyone else but this one was '
+                           'trying to submit it because I wanted to submit to the contest :) Tanks I hope the website is '
+                           'fixed PS: Jan you are cool please add my video',
+            'thumbnail': 'https://www.vidlii.com/usfi/thmp/tJluaH4BJ3v.jpg',
```

Comment:
No exact URLs.

Reply:
How should the matching be performed, only by considering the thumbnail's file extension (https://www.vidlii.com/*.jpg)?

Reply:
See another extractors.

Reply:
FIXED.